lib/ansible/modules/database/postgresql/postgresql_info.py

```diff
@@ -600,9 +610,35 @@ def collect(self, val_list=False):
 
         return self.pg_info
 
+    def get_pub_info(self):
+        """Get publication statistics."""
+        query = ("SELECT p.*, r.rolname AS ownername "
+                 "FROM pg_catalog.pg_publication AS p "
+                 "JOIN pg_catalog.pg_roles AS r "
+                 "ON p.pubowner = r.oid")
+
+        result = self.__exec_sql(query)
+
+        if result:
+            result = [dict(row) for row in result]
+        else:
+            return {}
+
+        publications = {}
+
+        for elem in result:
+            if not publications.get(elem['pubname']):
+                publications[elem['pubname']] = {}
+
+            for key, val in iteritems(elem):
+                if key != 'pubname':
+                    publications[elem['pubname']][key] = val
+
+        return publications
+
     def get_subscr_info(self):
         """Get subscription statistics and fill out self.pg_info dictionary."""
-        if self.cursor.connection.server_version < 10000:
+        if self.cursor.connection.server_version < 100000:
```

Comment:
Was this an error?

Reply:
yes:) definitely.
It doesn't work in PG versions < 10 (we have tests only for 10+ here).  I tested them manually with lower versions, found this, fixed

Reply:
Would it make sense to fix it in another PR (and backport)? Or was that feature also added for 2.10?

Reply:
yep, it hasn't been released yet

Reply:
Ah, perfect!